style.css

```diff
@@ -0,0 +1,246 @@
+@import url("https://fonts.cdnfonts.com/css/poppins");
+
+/* media query for mobile devices */
+@media only screen and (max-width: 375px) {
+  body {
+    /* header-background-mobile */
+
+    position: absolute;
+    width: 375px;
+    height: 718px;
+    background: #fff;
```

Comment:
- [x] I kindly 🙏🙏request that you ensure your project is responsive according to the project [requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/1_setup.md), 
- [x] specifically targeting mobile versions ranging from `375px to 768px.` By optimizing the design for different screen sizes, you will create a more user-friendly and visually appealing experience for your audience. 🥱🥱
- [x] Your attention to this detail will greatly enhance the overall quality of your project.
![res](https://github.com/MasterLeiva/DavidLeiva-Portfolio/assets/118761303/35405c2a-a681-43e3-bb83-a31439f33517)